lib/apruve/resources/invoice.rb

```diff
@@ -1,12 +1,21 @@
 module Apruve
   class Invoice < Apruve::ApruveObject
     attr_accessor :id, :order_id, :status, :amount_cents, :currency, :merchant_notes, :merchant_invoice_id, :shipping_cents,
-                  :tax_cents, :invoice_items, :payments, :created_at, :opened_at, :due_at, :final_state_at, :links
+                  :tax_cents, :invoice_items, :payments, :created_at, :opened_at, :due_at, :final_state_at, :issue_on_create, :links
 
     def self.find(id)
       response = Apruve.get("invoices/#{id}")
       Invoice.new(response.body)
     end
+    
+    def self.index(order_id)
```

Comment:
The semantics here don't match our web service API. Should either move this to Order, such as `Order#invoices_for(order_id)` or make this a true query action like `Invoice#index({query_params})`